### PR TITLE
clean up manage.py strings

### LINKED ISSUE #2454: add-admin returns confusing strings
# Bug

## Description

When adding the first admin user on a new SecureDrop install, it returns a confusing string
`./manage py add-admin`
Username: 
Note: _Journalist_ passwords are now autogenerated

## Steps to Reproduce

Add a new admin user on a fresh SecureDrop install

## Expected Behavior

Our script outputs administrator added to the user 

## Actual Behavior

Our script says we created a new journalist user not an administrator.

### PR DESCRIPTION
**Pull request by heartsucker**
## Status

Ready for review

## Description of Changes

Fixes #2454.

Fixes ambiguous language confusing you/they and admin/user/journalist.

### If you made changes to the app code:

- [x] Unit and functional tests pass on the development VM

### CODE REVIEW

File: securedrop/manage.py [hunk 1]
```diff
@@ -148,16 +148,16 @@ def _make_password():
 def _add_user(is_admin=False):
     username = _get_username()
 
-    print("Note: Journalist passwords are now autogenerated.")
+    print("Note: Passwords are now autogenerated.")
     password = _make_password()
-    print("This journalist's password is: {}".format(password))
+    print("This admin's password is: {}".format(password))
```
Comment (redshiftzero) [inline securedrop/manage.py]: Recall we also have a `manage.py` command for adding journalist accounts:

```py
    journalist_subp_a = subps.add_parser('add_journalist', help='^')
    journalist_subp_a.set_defaults(func=add_journalist)
```

So the messages here should vary on the value of kwarg `is_admin`.

File: securedrop/manage.py [hunk 2]
```diff
@@ -148,16 +148,17 @@ def _make_password():
...
+    print("This {}'s password is: {}"
+          .format('admin' if is_admin else 'journalist', password))
 
     is_hotp = _get_yubikey_usage()
     otp_secret = None
     if is_hotp:
         while True:
             otp_secret = raw_input(
-                'Please configure your YubiKey and enter the secret: ')
+                "Please configure this admin's YubiKey and enter the secret: ")
```
Comment (freddymartinez9) [inline securedrop/manage.py]: would this work if you add journalists only from the CLI?

Comment (redshiftzero) [inline securedrop/manage.py]: My comment [here](https://github.com/freedomofpress/securedrop/pull/2476#discussion_r146936772) also applies to this line. You can also just generically say "user" for simplicity if you prefer (either approach is fine for merge).

File: securedrop/manage.py [hunk 3]
```diff
@@ -178,7 +179,7 @@ def _add_user(is_admin=False):
                                                   exc_traceback)))
         return 1
     else:
-        print('User "{}" successfully added'.format(username))
+        print('Admin "{}" successfully added'.format(username))
```
Comment (redshiftzero) [inline securedrop/manage.py]: [also this line](https://github.com/freedomofpress/securedrop/pull/2476#discussion_r146936772)

**Pull request merged**